### DISCUSSION TITLE
Probably a stupid question, but why am I getting "not a valid identifier" errors?

### DISCUSSION THREAD

**Discussion by Mattncb** [abraunegg/onedrive#2493] [Q&A] [answered]
Hello Docker is working as non root but I always get the below error when I try to set up Onedrive Docker. 

Any idea what I'm doing wrong?
```
Status: Downloaded newer image for driveone/onedrive:edge
docker.io/driveone/onedrive:edge
admin@Opi5:~$ docker volume create onedrive_conf
onedrive_conf
admin@Opi5:~$ export "/home/admin/Onedrive"="${HOME}/OneDrive"                      
-bash: export: `/home/admin/Onedrive=/home/admin/OneDrive': not a valid identifier
```

**Comment by abraunegg** [comment, marked as answer]
@Mattncb 
> Any idea what I'm doing wrong?

You are essentially not following the documentation.

Where you are using this command:
```
admin@Opi5:~$ export "/home/admin/Onedrive"="${HOME}/OneDrive" 
```

The `"/home/admin/Onedrive"` is what is wrong. This needs to be changed to a variable that you use. Please re-read the documentation to help: https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md

**Reply by Mattncb** [reply]
I'm still not sure what I'm doing wrong?

The guide says 

Run the docker image with the commands below and make sure to change ONEDRIVE_DATA_DIR to the actual onedrive data directory on your filesystem that you wish to use (e.g. "/home/abraunegg/OneDrive").

and /home/admin/Onedrive is the correct data directory I wish  to use? (admin is the name of my user)

**Comment by abraunegg** [comment]
@Mattncb 
The documentation says this:
![image](https://github.com/abraunegg/onedrive/assets/4956234/1eb9abce-e484-4523-99c8-3c54d55425fe)

Specifically in your case, this should then read:
```
export  ONEDRIVE_DATA_DIR="/home/admin/Onedrive" 
```
